src/App.tsx

```diff
@@ -1,24 +1,21 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { ProductDetail } from "@pages/ProductDetail";
+import Products from "@pages/Products";
+import { BrowserRouter, Navigate, Route, Routes } from "react-router-dom";
+
+import "./App.scss";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="app">
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<Products />} />
+          <Route path="/product">
+            <Route path=":id" element={<ProductDetail />} />
+          </Route>
```

Comment:
сейчас получается так, что, если перейти на /product (без указания id), но отренедрится пустая страница. можно это разрулить, например, таким образом:
```suggestion
          <Route path="/product/:id"  element={<ProductDetail />} />
```

тогда при переходе на /product будет откидывать на "/"